packages/dialtone-tokens/themes/config.js

```diff
@@ -33,6 +43,17 @@ function _setStyleTag (id, content, rootNode) {
   }
 }
 
+
+/**
+ * Remove a style tag with the given id
+ */
+function _removeStyleTag (id, rootNode) {
+  const existingStyleTag = rootNode.querySelector('#' + id);
```

Comment:
We should have an existence check on rootNode.

Probably just `rootNode?.querySelector('#' + id);` is fine